android/java/org/chromium/chrome/browser/BraveRewardsSiteBannerActivity.java

```diff
@@ -73,13 +72,16 @@ protected void onCreate(Bundle savedInstanceState) {
         currentTabId_ = IntentUtils.safeGetIntExtra(getIntent(), TAB_ID_EXTRA, -1);
         mIsMonthlyContribution =
                 IntentUtils.safeGetBooleanExtra(getIntent(), IS_MONTHLY_CONTRIBUTION, false);
-        if (savedInstanceState == null) {
-            mBraveRewardsNativeWorker.GetPublisherBanner(
-                    mBraveRewardsNativeWorker.GetPublisherId(currentTabId_));
-        }
+        mBraveRewardsNativeWorker.GetPublisherBanner(
+                mBraveRewardsNativeWorker.GetPublisherId(currentTabId_));
```

Comment:
the calls to native have to go inside `finishNativeInitialization`

Reply:
updated

Reply:
```
        mBraveRewardsNativeWorker = BraveRewardsNativeWorker.getInstance();
        mBraveRewardsNativeWorker.AddObserver(this);
```
that part is also doing calls to native libs, move it to `finishNativeInitialization` as well

Reply:
updated